### PR TITLE
Add role_name to cloud account resource

### PR DESCRIPTION
**Pull request by aq17**

### CODE REVIEW

**Review by nkraemer-sysdig — APPROVED** [review]

File: website/docs/r/sysdig_secure_cloud_account.md [hunk 1]
```diff
@@ -20,18 +20,21 @@ resource "sysdig_secure_cloud_account" "sample" {
...
+* `account_id` - (Required) The unique identifier of the cloud account. e.g. for AWS: `123456789012`,
 
 * `cloud_provider` - (Required) The cloud provider in which the account exists. Currently supported providers are `aws`, `gcp` and `azure`
 
 * `alias` - (Optional) A human friendly alias for `account_id`.
 
-* `role_enabled` - (Optional) Whether or not a role with the name `SysdigCloud` is provisioned withing this account, that Sysdig has permission to AssumeRole in order to run Benchmarks. Default: `false`.
+* `role_enabled` - (Optional) Whether or not a role is provisioned withing this account, that Sysdig has permission to AssumeRole in order to run Benchmarks. Default: `false`.
+
+* `role_name` - (Optional) The name of the role Sysdig will have permission to AssumeRole if `role_enaled` is set to `true`. Default: `SysdigCloud`.
```
Comment (wideawakening) [inline website/docs/r/sysdig_secure_cloud_account.md:37]: Default would be `SysdigCloudBench`, right?

LGTM! 
This PR will enable us to test better the tf-aws module, as we can avoid RoleName collision, thx!

**Comment by nestorsalceda** [comment]
Please @aq17 make sure you run the terraform format :)

Thank you!

File: website/docs/r/sysdig_secure_cloud_account.md (hunk 1 above)
Comment (aq17) [reply website/docs/r/sysdig_secure_cloud_account.md]: Updated!

**Comment by nestorsalceda** [comment]
Hey @aq17 I meant the `make fmt` in order to fix the build:

https://github.com/sysdiglabs/terraform-provider-sysdig/pull/121/checks?check_run_id=3546799195

Thank you man! :)

**Comment by nkraemer-sysdig** [comment]
@nestorsalceda is there anything else to do before this can be merged? The tests are failing since this is a fork

**Pull request merged**